src/test/java/com/github/fishio/logging/TestTxtFileHandler.java

```diff
@@ -23,22 +21,16 @@
 public class TestTxtFileHandler extends TestIHandler {
```

Comment:
*Auto-generated comment*

    
    This class has too many methods, consider refactoring it.